src/api/preferencesApi.ts

```diff
@@ -0,0 +1,23 @@
+import { PersonPreferences } from '../types/personPreferences.types';
+import { PersonPreferencesApiPath } from './apiPaths';
+import { authenticatedApiRequest2 } from './apiRequest';
+
+export const fetchPromotedPublicationsById = async (id: string) => {
+  const getPromotedPublications = await authenticatedApiRequest2<PersonPreferences>({
+    url: `${PersonPreferencesApiPath.PersonPreferences}/${id}`,
+  });
+
+  return getPromotedPublications.data;
+};
+
+export const updatePromotedPublications = async (personIdentifier: string, promotedPublications: string[]) => {
+  const updatePromotedPublication = await authenticatedApiRequest2<PersonPreferences>({
+    url: `${PersonPreferencesApiPath.PersonPreferences}/${personIdentifier}`,
```

Comment:
fant vel ut at dette faktisk skulle være id (URI)
```suggestion
export const updatePromotedPublications = async (personId: string, promotedPublications: string[]) => {
  const updatePromotedPublication = await authenticatedApiRequest2<PersonPreferences>({
    url: `${PersonPreferencesApiPath.PersonPreferences}/${personId}`,
```

Reply:
syns vel egentlig at identifier hadde gitt mer mening jeg da, men la oss ikke bruker mer tid på det 😅